<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshAdaptWidget</class>
 <widget class="QWidget" name="MeshAdaptWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="solverStudyNodeFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Error indicator:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QmitkDataStorageComboBox" name="errorIndicatorNodeComboBox">
          <property name="toolTip">
           <string>Scalar array representing the error</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="meshNodeName">
          <property name="toolTip">
           <string>Name of the mesh</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Mesh:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="adaptationParametersFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0" columnstretch="0,1">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Error reduction factor:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="errorReductionFactorSpinBox">
        <property name="toolTip">
         <string>Target average error reduction factor</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.001000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Minimum edge size:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="minimumEdgeSizeSpinBox">
        <property name="toolTip">
         <string>Minimum edge size after adaptation</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.001000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Maximum edge size:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="maximumEdgeSizeSpinBox">
        <property name="toolTip">
         <string>Maximum edge size after adaptation</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.001000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="adaptMeshButton">
       <property name="text">
        <string>Adapt mesh</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/SolverSetup.qrc">
         <normaloff>:/contourModeling/Icon_MeshAdaptView.png</normaloff>:/contourModeling/Icon_MeshAdaptView.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="cancelAdaptMeshButton">
       <property name="toolTip">
        <string>Cancel the current mesh adaptation operation</string>
       </property>
       <property name="text">
        <string>Cancel mesh</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/SolverSetup.qrc">
         <normaloff>:/contourModeling/icons/stop.png</normaloff>:/contourModeling/icons/stop.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>adaptationParametersFrame</zorder>
  <zorder>solverStudyNodeFrame</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">QmitkDataStorageComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>meshNodeName</tabstop>
  <tabstop>errorIndicatorNodeComboBox</tabstop>
  <tabstop>errorReductionFactorSpinBox</tabstop>
  <tabstop>minimumEdgeSizeSpinBox</tabstop>
  <tabstop>maximumEdgeSizeSpinBox</tabstop>
  <tabstop>adaptMeshButton</tabstop>
  <tabstop>cancelAdaptMeshButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/SolverSetup.qrc"/>
  <include location="../../../uk.ac.kcl.VesselMeshing/resources/VesselMeshing.qrc"/>
 </resources>
 <connections/>
</ui>
